tests/Functional/BaseTestCase.php

```diff
@@ -46,7 +46,7 @@ protected function getContainer()
         return $this->container;
     }
 
-    protected function getDocumentManager()
+    protected function getManager()
```

Comment:
Its the document manager. I think `manager` can be assumed to refer to the document manager in the document manager package.